src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -266,7 +266,7 @@ struct sdm_sensor_info
     }
 
     if (path.empty())
-      return result_type();
+      throw xrt_core::query::sysfs_error("sysfs path not found");
```

Comment:
Could you help the future developer an let us know "what path" was not found.   

I ask for this exception basically states....I found an issue, but I am not going to tell you what it is.

Reply:
Hi @rozumx please let me know if below error message is ok here, thank you.

> throw xrt_core::query::sysfs_error("target hwmon sysfs path /sys/bus/pci/devices/<bdf>/hwmon/hwmon*/ not found");

Reply:
This will work.